docs/api/api.yml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.0
+info:
+  title: Download Service API
+  version: 1.1.2
+  description: |
+    This document specifies the Download Service API. It provides a direct interface to download files from the archive.
+    This API implements the operations `GET` on the paths `/` and `/ark:/:NAAN/:uuid/*`.
+    The NAAN represents the configured 'name assigning authority number' which for DANS is 73189.
+    It is not neccessary for this service to supply the bag store the bag is stored in.
+
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0
+
+servers:
+  - url: 'http://localhost:20160'
+  - url: 'http://test.dans.knaw.nl:20160'
+
+paths:
+  /:
+    get:
+      summary: Returns a description of the service.
+      description:
+        This is the starting place for navigating the Download Service. It returns a description of the service.
+      responses:
+        200:
+          description: Description of the service
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: A plain text, human readable description of the service
+                example: Download Service is running.
+        500:
+          $ref: '#/components/responses/Unavailable'
+
+  /ark:/{NAAN}/{ItemId}:
+    get:
+      summary: Returns an output stream of the file specified by the ItemId.
+      description: |
+        An item is either a directory or file in a bag. The item which is fetched is specified by the item-id, which is the bag-id followed by the local path of the file in the bag.
```

Comment:
It can only download single files, not directories.
Also it is not just any file in the bag, but only those in the payload of the bag that can be downloaded through here.

Reply:
I believe i did specify files earlier in the documentation. This line is just the definition of `Item-Id`, which is used in more projects. I wanted to keep this definition consistent across projects, so i kept it like this. Do you think it makes it unclear? 

(so, an item-id can refer to a bag or directory, but only item-id's of payload files are accepted)

Reply:
My reasoning was that you first start talking about an `item` which can be a **directory** or a **file**. Afterwards you state that you can't download a directory (yet). That sounds a bit weird to me.

I see the problem, though... you want to be both precise in your definition of `item` and clarify the restrictions we currently have for this service. Let's keep it as is for now and take another look once we implement downloading a whole bag or directories in a bag.

Reply:
* If currently we can only download files, that's what the documentation should say. 
* Just refer to [the definition of file-id](https://dans-knaw.github.io/easy-bag-store/03_definitions.html#item). Your description does not take into account the percent-encoding rules.

Reply:
I updated the description to refer to the file-id instead of the item-id.